tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Various learning rate decay functions."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.framework import ops, dtypes
```

Comment:
Fixed, thanks.

On Mon, Mar 28, 2016, 12:39 PM Vijay Vasudevan <notifications@github.com>
wrote:

> In tensorflow/python/training/initial_learning_rate_decay.py
> <https://github.com/tensorflow/tensorflow/pull/1512#discussion_r57620742>:
>
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +# ==============================================================================
> > +
> > +"""Various learning rate decay functions."""
> > +from __future__ import absolute_import
> > +from __future__ import division
> > +from __future__ import print_function
> > +
> > +from tensorflow.python.framework import ops, dtypes
>
> I think we prefer importing one module per line (like 22 and 23)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/1512/files/0aa1e5d124681a94dfb3ec6b65ce86f8410f4eb3#r57620742>
>